src/views/Deposit/components/EnterAmount/index.tsx

```diff
@@ -29,8 +116,12 @@ export const EnterAmount: React.FC = () => {
         <StyledHeaderRight>
           <StyledLine />
           <div id={'wallet-balance'}>
-            <P2 color={colors.white}>Wallet balance: 2345.926562 wPOKT</P2>
-            <StyledMaxButton>
+            <P2 color={colors.white}>
+              {wpoktBalance
+                ? `Wallet balance: ${bnToDec(new BigNumber(wpoktBalance))} wPOKT`
```

Comment:
Same—can have overflow issues

Reply:
Fixed in 6cd632d7c520d20ea5f02a83b1ea1b4da83ad696